pilot/pkg/xds/debug.go

```diff
@@ -1112,9 +1114,15 @@ func (p jsonMarshalProto) MarshalJSON() ([]byte, error) {
 }
 
 // writeJSON writes a json payload, handling content type, marshaling, and errors
-func writeJSON(w http.ResponseWriter, obj interface{}) {
+func writeJSON(w http.ResponseWriter, obj interface{}, req *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	b, err := config.ToPrettyJSON(obj)
+	var b []byte
+	var err error
+	if req.URL.Query().Has("format") {
```

Comment:
rename to `pretty`?

Reply:
I have no strong preference. Will change